### DISCUSSION TITLE
Diagnostic Sensors

### DISCUSSION THREAD

**Discussion by coreywillwhat** [audiconnect/audi_connect_ha#340] [Ideas]
Curious the thoughts from devs/community on moving some sensors to the diagnostic `entity_category`. Currently the sensor list under the device is quite long. This could help organizing/quickly finding information.

#### Potential Sensors to Move to Diagnostic:
- Oil Change Distance
- Oil Change Time
- Mileage
- Service Inspection Distance
- Service Inspection Time

#### Maybe These?
- Car Type
- Model
- Parking Light
- Plug LED Color

**Comment by Kolbi** [comment]
primary_engine_type ?
every single window / door ? => but combined status (all doors / all windows) as normal sensor?

**Reply by coreywillwhat** [reply]
secondary_engine_type

**Comment by coreywillwhat** [comment]
Here is where I'm at. Would love both your input on this @Kolbi @cdnninja , as well as anyone else who is taking a look.
![image](https://github.com/audiconnect/audi_connect_ha/assets/104224685/28c29c2f-7349-48ce-9369-899ebbe3c9bc)
![image](https://github.com/audiconnect/audi_connect_ha/assets/104224685/5ce977f9-efb3-4282-bb7c-4c6f43cf95f8)

**Reply by cdnninja** [reply]
Looks good to me.  Can always adjust based on feedback.

**Reply by Kolbi** [reply]
Yeah looks good. What about "Plug Lock State"?
